file_with_secrets.txt

```diff
@@ -1,8 +1 @@
------BEGIN PRIVATE KEY-----
-BASE64 ENCODED DATA
------END PRIVATE KEY-----
-
-xoxb-111222333444-aaabbbcccdddeeefffggghhh
-xoxb-111222333444-aaabbbcccdddeeefffgggzzz
-
-AKIAAAAAAAAAAAAAAAAA
+no secrets here
```

Comment:
`push` trigger checks latest commit (current) and `gitleaks` detects nothing in this case as it analysis just 1 commit (where we removed secrets and no leakage).
![image](https://user-images.githubusercontent.com/3036347/96785426-75d9f400-13f7-11eb-9ef7-8f549b288f34.png)



This won't make the `pull_request` check happy since commits history is not rewritten.
It's expected to fail as I would like to leave commits history for future reference.

![image](https://user-images.githubusercontent.com/3036347/96785162-14198a00-13f7-11eb-86e4-e71f43fb388a.png)
![image](https://user-images.githubusercontent.com/3036347/96785379-6195f700-13f7-11eb-8a9d-cc7fc4101e12.png)